features/step_definitions/search_functionality_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^that i am on the landing page$/) do
+  visit root_path
+end
+
+Given(/^there is a field "([^"]*)"$/) do |q|
+  find_field('search_field').value
```

Comment:
You can ignore this comment. I will update the config to not have this in future